tests/test_pipreqs.py

```diff
@@ -559,6 +597,32 @@ def test_parse_requirements_handles_file_not_found(self, exit_mock):
 
             self.assertEqual(printed_text, "File xpto was not found. Please, fix it and run again.")
 
+    def test_ignore_notebooks(self):
+        """
+        Test if notebooks are ignored when the scan-notebooks parameter is False
+        """
+        pipreqs.init(
+            {
+                "<path>": self.project_with_notebooks,
+                "--savepath": None,
+                "--use-local": None,
+                "--force": True,
+                "--proxy": None,
+                "--pypi-server": None,
+                "--print": False,
+                "--diff": None,
+                "--clean": None,
+                "--mode": None,
+                "--scan-notebooks": False,
+            }
+        )
+        assert os.path.exists(self.requirements_notebook_path) == 1
```

Comment:
where is requirements_notebook_path defined? cant find it for some reason

Reply:
Fixed.